x-pack/plugins/uptime/server/lib/saved_objects/migrations.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectUnsanitizedDoc } from 'src/core/server';
+import { DynamicSettings } from '../../../common/runtime_types';
+
+export const add820Indices = (doc: SavedObjectUnsanitizedDoc<DynamicSettings>) => {
+  const { heartbeatIndices } = doc.attributes;
+
+  const indicesArr = heartbeatIndices.split(',');
```

Comment:
can we also add a condition to check if the field is not present? or if it's empty. 

In case of migrations, we always assume the worst case scenerios and don't rely on typings. 

Because one bag migration or bad object can impact kibana restart.